service-registry/src/main/java/org/apache/servicecomb/serviceregistry/consumer/MicroserviceVersions.java

```diff
@@ -76,6 +77,18 @@ public MicroserviceVersions(AppManager appManager, String appId, String microser
     appManager.getEventBus().register(this);
   }
 
+  public void updateLastAccessTime() {
+    this.lastAccessedTime = System.currentTimeMillis();
+  }
+
+  public boolean isValidated() {
```

Comment:
when will a MicroserviceVersions created, but never used?

Reply:
when microserivce not exists or for the first time some unexpected sc problems happens